hw/ip/rv_core_ibex/lint/rv_core_ibex.waiver

```diff
@@ -88,6 +88,10 @@ waive -rules HIER_BRANCH_NOT_READ -location {ibex_decoder.sv ibex_compressed_dec
       -comment "These signals are only used for assertions inside these two modules"
```

Comment:
there is a `ONE_BRANCH` waiver at the top of this file that can be removed since we've added this to a common waiver file now.